tests/integration/test_ethernet.py

```diff
@@ -0,0 +1,93 @@
+import os
+import pytest
+import subprocess
+import sys
+
+VETH_NAME = "testeth"
+
+if sys.version_info >=(3, 3):
+    from unittest import mock
+else:
+    import mock
+
+def configure_network(connections):
+    parentdir = os.path.normpath(os.path.join(os.path.dirname(__file__), "../../"))
+    with mock.patch.object(
+        sys,
+        "path",
+        [parentdir, os.path.join(parentdir, "module_utils/network_lsr")] + sys.path,
+    ):
+        with mock.patch.dict(
+            "sys.modules",
+            {
+                "ansible": mock.Mock(),
+                "ansible.module_utils": __import__("module_utils"),
+            },
+        ):
+            import library.network_connections as nc
+
+    run_env = nc.RunEnvironment()
```

Comment:
Please create a class like "TestRunEnvironment" that inherits from "RunEnvironment" and implements the necessary methods like the "AnsibleRunEnvironment" does it in the library.

Reply:
I fixed it, thanks! I did this like the first way you mentioned. Although I might not be right, for me it sounds wrong to add classes or functions to the library as long as they are only going to be used for tests. Maybe it would be useful to make a "utils" module for tests in a future though.